### PR TITLE
Release WZDx v4.2

### PR DESCRIPTION
**Pull request by mark-mockett**
## New Functionality
- Add `impacted_cds_curb_zones` property to the [WorkZoneRoadEvent](https://github.com/usdot-jpo-ode/wzdx/blob/release/v4.2/spec-content/objects/WorkZoneRoadEvent.md) object to allow indicating if any [Curb Data Specification](https://github.com/openmobilityfoundation/curb-data-specification) curb zones are impacted by the work zone using new [CdsCurbZonesReference](https://github.com/usdot-jpo-ode/wzdx/blob/release/v4.2/spec-content/objects/CdsCurbZonesReference.md) object.
- Add `work_zone_type` property to the [WorkZoneRoadEvent](https://github.com/usdot-jpo-ode/wzdx/blob/release/v4.2/spec-content/objects/WorkZoneRoadEvent.md) object to allow indicating if a work zone road event represents a planned moving operation, an active moving operation, or a standard static work zone. 
- Add the following values to the [Direction](https://github.com/usdot-jpo-ode/wzdx/blob/release/v4.2/spec-content/enumerated-types/Direction.md) enumerated type:
    - `inner-loop`
    - `outer-loop`
- Add `velocity_kph` property to the [FieldDeviceCoreDetails](https://github.com/usdot-jpo-ode/wzdx/blob/release/v4.2/spec-content/objects/FieldDeviceCoreDetails.md) object to allow indicating the velocity of a field device.
- Add `work-truck-with-lights-flashing` to the [MarkedLocationType](https://github.com/usdot-jpo-ode/wzdx/blob/release/v4.2/spec-content/enumerated-types/MarkedLocationType.md) enumerated type.

## Refactoring
No properties were deprecated or removed in WZDx v4.2.

## Cleanup
- Expand the description of the `geometry` property on the [RoadEventFeature](https://github.com/usdot-jpo-ode/wzdx/blob/release/v4.2/spec-content/objects/RoadEventFeature.md) object to clarify how geometry should be used to represent a road event.
- Update the description of the `open` and `closed` enumerations of the [LaneStatus](https://github.com/usdot-jpo-ode/wzdx/blob/release/v4.2/spec-content/enumerated-types/LaneStatus.md) enumerated type.
- Update the description of the [FlashingBeacon](https://github.com/usdot-jpo-ode/wzdx/blob/release/v4.2/spec-content/objects/FlashingBeacon.md) object to clarify that it should only be used as a flashing warning beacon mounted on a temporary traffic control device.

### CODE REVIEW

**Review by mollybehan — APPROVED** [review]

**Review by sknick-iastate — APPROVED** [review]

**Review by jlarsonOmahaNE — APPROVED** [review]

**Review by mollybehan — DISMISSED** [review]

**Review by sknick-iastate — APPROVED** [review]

File: spec-content/objects/Relationship.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Relationship Object
 The `Relationship` object is used to identify both sequential and hierarchical relationships between a road events and other entities. For example, a relationship can be used to link multiple road events to a common "parent", such as a project or phase, or identify a sequence of road events that make up a larger work zone.
 
-#### *All properties that use the Relationship object are deprecated as of WZDx v4.1 and along with those properties it will be removed from the specification in the next major release.*
+#### *All properties that use the Relationship object are deprecated as of WZDx v4.2 and along with those properties it will be removed from the specification in the next major release.*
```
Comment (j-d-b) [inline spec-content/objects/Relationship.md:4]: I think this should stay as v4.1 since that is when they were deprecated

**Review by j-d-b — COMMENTED** [review]
Great typo fixes, just one comment, otherwise looks great.

**Review by j-d-b — APPROVED** [review]

File: spec-content/objects/Relationship.md (hunk 1 above)
Comment (j-d-b) [reply spec-content/objects/Relationship.md]: Changed.

Comment (mark-mockett) [reply spec-content/objects/Relationship.md]: thanks @j-d-b  - just added a commit to update the other notes on EventStatus, TimeVerification, and SpatialVerification

**Review by j-d-b — APPROVED** [review]

**Review by sknick-iastate — APPROVED** [review]

**Review by jlarsonOmahaNE — APPROVED** [review]

**Pull request merged**